src/storage/src/hummock/state_store.rs

```diff
@@ -87,14 +99,23 @@ impl HummockStorage {
         validate_epoch(pinned_version.safe_epoch(), epoch)?;
 
         // check epoch if lower mce
-        let read_version_tuple = if epoch <= pinned_version.max_committed_epoch() {
-            // read committed_version directly without build snapshot
-            (Vec::default(), Vec::default(), (**pinned_version).clone())
-        } else {
-            // TODO: use read_version_mapping for batch query
-            let read_version_vec = vec![self.read_version.clone()];
-            read_filter_for_batch(epoch, table_id, &key_range, read_version_vec)?
-        };
+        let read_version_tuple: (Vec<ImmutableMemtable>, Vec<SstableInfo>, CommittedVersion) =
```

Comment:
Seems this method is a duplicate to the code in `get`. Can we extract them to a common method?